src/codegen/function_body_chunk.rs

```diff
@@ -367,6 +346,17 @@ impl Builder {
         }
     }
 
+    fn is_uninitialized_var(
+        &self,
+        name: &String,
+        uninitialized_vars: &mut [(String, bool)],
```

Comment:
```suggestion
        uninitialized_vars: &[(String, bool)],
```

Doesn't have to be mutable